src/transforms/reduce/transform.rs

```diff
@@ -261,7 +325,9 @@ impl TaskTransform<Event> for Reduce {
             flush_period,
             |me: &mut Box<Reduce>, event, emitter: &mut Emitter<Event>| {
                 // called for each event
-                me.transform_one(emitter, event);
+                if let Err(error) = me.transform_one(emitter, event) {
+                    error!(%error, rate_limit = 30)
```

Comment:
Should this be emitting an "internal event" with structured data and a metric?

Reply:
Would something like `vector_lib::registered_event! (ReduceEventsDropped => {` sense here?

Reply:
Created src/internal_events/reduce.rs and emitting it inside `add_event(..)`.